thoth/python/source.py

```diff
@@ -414,38 +388,42 @@ def provides_package_version(self, package_name: str, package_version: str) -> b
             return False
 
     @lru_cache(maxsize=10)
-    def get_package_data(self, package_name: str, package_version: str, with_included_files: bool = False) -> list:
-        """Get information about release hashes and symbols available in this source index."""
+    def get_package_hashes(
```

Comment:
Why the change of the function name/doc line?

Reply:
I checked for the function `get_package_data` across whole github, it didn't seem be used anywhere.
so merged it in function `get_package_hashes`

Reply:
Ahh got it